src/custom-elements/src/components/search-input/search-input.tsx

```diff
@@ -266,6 +270,7 @@ export class SearchInput {
         position="bottom"
         prevent-auto-hide
         fluid
+        visible-items={this.visibleItems && this._isAutocomplete() ? this.visibleItems : null}
```

Comment:
Can we better use a constant outside to get the value of visibleItems? 

`const visibleItems = this.visibleItems ?? null;`

Also, there is no need to check if it is **autocomplete** or not as the dropdown is only generated if it is already autocomplete (line 349).

```suggestion
        visible-items={visibleItems}
```